### PR TITLE
[ADD][11.0] - Adding module project_agile_scrum

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello,

Module ``project_agile_scrum`` which extends module [project-agile](https://github.com/OCA/project-agile/pull/4) enables you to manage your projects by using agile scrum methodology.

Kind Regards,
Petar

### CODE REVIEW

File: project_agile_scrum/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_agile_scrum/README.rst]: see https://github.com/OCA/web/pull/881#pullrequestreview-99094143

File: project_agile_scrum/models/project_agile_scrum_sprint.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            hrs = days * float(rec.default_hrs)
+            hrs_int = int(hrs)
+            rec.sprint_length_hrs = hrs_int
+
+    @api.model
+    @api.returns('self', lambda value: value.id)
+    def create(self, vals):
+        if not self.env.user.team_id:
+            raise exceptions.ValidationError(_(
+                "You have to be part of an agile team in order to "
```
Comment (elicoidal) [inline project_agile_scrum/models/project_agile_scrum_sprint.py]: agile -> Agile ?

File: project_agile_scrum/models/project_agile_team.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    completed_sprint_count = fields.Integer(
+        string="Future Sprint Count",
+        compute="_compute_completed_sprint_count",
+    )
+
+    default_sprint_length = fields.Selection(
+        selection=[
+            ('1', 'One Week'),
+            ('2', 'Two Weeks'),
+            ('3', 'Tree Weeks'),
```
Comment (elicoidal) [inline project_agile_scrum/models/project_agile_team.py]: Three

File: project_agile_scrum/views/project_agile_team_views.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <field name="arch" type="xml">
+            <field name="image_small" position="after">
+                <field name="master_id"/>
+                <field name="type"/>
+                <field name="default_sprint_length"/>
+                <field name="velocity"/>
+            </field>
+
+
+            <li name="email" position="before">
```
Comment (elicoidal) [inline project_agile_scrum/views/project_agile_team_views.xml:58]: Could you format a bit in order to improve readability?

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details.
Missing UT and demo data
Not tests

File: project_agile_scrum/README.rst (hunk 1 above)
Comment (pnajman-modoolar) [reply project_agile_scrum/README.rst]: Fixed

File: project_agile_scrum/models/project_agile_scrum_sprint.py (hunk 2 above)
Comment (pnajman-modoolar) [reply project_agile_scrum/models/project_agile_scrum_sprint.py]: Changed

File: project_agile_scrum/models/project_agile_team.py (hunk 3 above)
Comment (pnajman-modoolar) [reply project_agile_scrum/models/project_agile_team.py]: Fixed

File: project_agile_scrum/views/project_agile_team_views.xml (hunk 4 above)
Comment (pnajman-modoolar) [reply project_agile_scrum/views/project_agile_team_views.xml:58]: I hope it's ok now ...

**Pull request closed without merging**